app/controllers/match.js

```diff
@@ -0,0 +1,52 @@
+// require list
+//const slug = require("slug");
+//const mongoose = require("mongoose");
+
+// display the user people that can be matched with
+// items needed from database: User image, name, city
+// for each person in database
+
+function showUser() {
+    //total users in database
+    const users = 10; //users.length

```

Comment:
You can't get the users.length if you've never defined the array of users in the first place. So what you need to do is set a `const users` to match `User.find()` so you get all the users.